chef/cookbooks/hyperv/recipes/openstack_install_nova.rb

```diff
@@ -24,3 +24,14 @@
   EOH
   not_if { ::File.exists?("#{node[:openstack][:nova][:installed]}") }
 end
+
+ruby_block "ensure correct python path in shebang" do
```

Comment:
The only thing I can think about should be another resource at https://github.com/crowbar/crowbar-core/tree/master/chef/cookbooks/utils/providers